### DISCUSSION TITLE
Add word-break: keep-all utility

### DISCUSSION THREAD

**Discussion by idl3** [tailwindlabs/tailwindcss#6273] [Ideas]
Recently needed to use word-break: keep-all and noticed that it was not included.
Specs for reference: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Any work on this started yet? If not I could take a stab at adding this in :)

Was thinking of using `break-keep` to keep to the same prefix.

**Comment by myriky** [comment]
any updates?

**Comment by greenuns** [comment]
god, what a useless library
